### PR TITLE
ui: fix language setting on the public interface

### PR DESCRIPTION
**Pull request by Garfield-fr**
* Fixes the current language on public interface.
* Improves center the text above thumbnail on document brief view.
* Moves the document availability at the end of the metadata of the
  document brief view.
* Closes rero/rero-ils#1391.

Co-Authored-by: Bertrand Zuchuat <bertrand.zuchuat@rero.ch>

## Code review check list

- [ ] Commit message template compliance.
- [ ] Commit message without typos.
- [ ] File names.
- [ ] Functions names.
- [ ] Functions docstrings.
- [ ] Unnecessary commited files?
- [ ] Cypress tests successful?

### CODE REVIEW

**Review by iGormilhit — CHANGES_REQUESTED** [review]
Commit message:

```
ui: fix language setting on the public interface

* Fixes the current language on public interface.
* Improves center the text above thumbnail on document brief view.
* Moves the document availability at the end of the metadata of the
  document brief view.
* Closes rero/rero-ils#1391.
```

**Review by iGormilhit — APPROVED** [review]
Commit message approved.

File: projects/public-search/src/app/app-initializer.service.ts [hunk 1]
```diff
@@ -31,16 +33,36 @@ export class AppInitializerService {
    */
```
Comment (sebastiendeleze) [inline projects/public-search/src/app/app-initializer.service.ts:33]: Doc for params is missing

**Review by sebastiendeleze — APPROVED** [review]

File: projects/public-search/src/app/app-initializer.service.ts [hunk 2]
```diff
@@ -28,19 +30,42 @@ export class AppInitializerService {
...
+        this.initTranslateService();
+      });
       this._userService.load();
       this._routeCollectionService.load();
       resolve(true);
     });
   }
+
+  /** Initialize Translate Service */
+  private initTranslateService(): void {
```
Comment (jma) [inline projects/public-search/src/app/app-initializer.service.ts:59]: can be one day in the shared or ng-core

**Review by jma — APPROVED** [review]

File: projects/public-search/src/app/app-initializer.service.ts (hunk 2 above)
Comment (Garfield-fr) [reply projects/public-search/src/app/app-initializer.service.ts:59]: For the next refactoring ;-)

**Pull request merged**